Dockerfile

```diff
@@ -0,0 +1,43 @@
+# Start with fresh Unbuntu
+FROM ubuntu:14.04
+
+# Install dependencies
+RUN apt-get update && \
+DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends \
+supervisor \
+git \
+apache2 \
+mysql-server \
+php5-cli php5-curl php5-mysql php5-gd php5-intl \
+libapache2-mod-php5 \
+imagemagick \
+curl
+
+EXPOSE 80 3306
```

Comment:
Let's set the port to diffrent one 8000, 9000 etc. If it will be set to 80 and some users will already have vhost defined under localhost:80 it will colidate.